src/browser-connection/local.js

```diff
@@ -1,19 +1,11 @@
 import { open as openBrowser, close as closeBrowser } from 'testcafe-browser-natives';
 import BrowserConnection from './index';
-import COMMAND from './command';
```

Comment:
check the `browser-connection\command.js` file. The `COMMAND.close` command is not used yet